ui/src/app/machines/views/MachineList/MachineListHeader/ActionFormWrapper/DeployForm/DeployForm.test.js

```diff
@@ -185,6 +185,101 @@ describe("DeployForm", () => {
     ]);
   });
 
+  it("can deploy with user data", () => {
```

Comment:
we could try to be consistent about calling this `user-data` to make it clear that it's the cloud-init user-data, distinct from an actual user's data.

Reply:
Done.